core/cache/package_test.go

```diff
@@ -23,17 +23,21 @@ func TestPackage(t *testing.T) {
 type BaseSuite struct {
 	jujutesting.IsolationSuite
 
+	Changes chan interface{}
```

Comment:
Why have these exported? They don't need the capital letter, so what is the rationale of using one?

Reply:
These were moved into the `BaseSuite` from the `ControllerSuite`. `BaseSuite` is exported from the `cache` package, while `ControllerSuite` is still in `cache_test`.